tmt/steps/execute/upgrade.py

```diff
@@ -312,8 +335,39 @@ def _perform_upgrade(
                 self._run_discover_upgrade()
                 # Pass in the path-specific env variables
                 extra_environment = plan.environment
+
+            dependencies: list[tmt.base.DependencySimple] = []
+            dependencies_recommend: list[tmt.base.DependencySimple] = []
```

Comment:
What about calling these two `required_packages` and `recommended_packages` to make it clear at the first sight?

Reply:
renamed

Reply:
Thanks.